tools/geo_converters/proc_gdal/proc_boundary_examples/data_proc_boundary_ca_50km.txt

```diff
@@ -0,0 +1,51 @@
+Running on worker USER = @K
```

Comment:
tools/geo_converters/proc_gdal/proc_boundary_examples/data_proc_boundary_ca_50km.txt and tools/geo_converters/proc_gdal/proc_boundary_examples/data_proc_boundary_us_50km.txt look like logs of program execution. Why store them in git?